### PR TITLE
Migrate postcss-focus-visible

### PR DESCRIPTION
**Pull request by romainmenke**
- converted to typescript as it had type defs
- uses a selector parser instead of a regexp.

----

Also updated `repository` field in postcss-focus-within

### CODE REVIEW

File: plugins/postcss-focus-visible/package.json [hunk 1]
```diff
@@ -4,97 +4,40 @@
   "description": "Use the :focus-visible pseudo-selector in CSS",
   "author": "Jonathan Neal <jonathantneal@hotmail.com>",
   "license": "CC0-1.0",
-  "repository": "csstools/postcss-focus-visible",
-  "homepage": "https://github.com/csstools/postcss-focus-visible#readme",
-  "bugs": "https://github.com/csstools/postcss-focus-visible/issues",
+    "homepage": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-focus-visible#readme",
```
Comment (Antonio-Laguna) [inline plugins/postcss-focus-visible/package.json:7]: [nit]

Comment (romainmenke) [inline plugins/postcss-focus-visible/package.json:7]:
```suggestion
  "homepage": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-focus-visible#readme",
```

Comment (romainmenke) [reply plugins/postcss-focus-visible/package.json]: Thank you!

**Review by Antonio-Laguna — APPROVED** [review]
Great one!

**Pull request merged**